docs/usage/configuration-options.md

```diff
@@ -10,16 +10,20 @@ Any config you define applies to the whole repository (e.g. if you have a monore
 
 You can store your Renovate configuration file in one of the following locations:
 
+- `renovate.json`
```

Comment:
> and the lack of formatting for this list items makes it much harder to read.

How about changing it into a numbered list? For example:

```markdown
1. `renovate.json`
1. `renovate.json5`
1. `.github/renovate.json`
```

Which looks like this:

1. `renovate.json`
1. `renovate.json5`
1. `.github/renovate.json`